CONTRIBUTING.md

```diff
@@ -6,6 +6,13 @@ this case, the [Apache Software License v2](LICENSE).
 
 ## Preparing your development environment
 
+Sygnal depends on the `psycopg2` database adapter for PostgreSQL. 
+It is recommended that you install `libq-dev python3-dev build-essential`. For example on Debian/Ubuntu these can be installed with:
```

Comment:
Thank you for the update -- I think this still needs to be wrapped at 80 characters (your editor should have a setting somewhere that does this automatically)

Otherwise, I'd suggest tweaking the wording very slightly to be a little more precise:

```diff
- It is recommended that you install `libq-dev python3-dev build-essential`.
+ You may need to install development headers for Python and libpq.
```

With those resolved we should be good to go :)